torch/distributions/binomial.py

```diff
@@ -77,6 +77,11 @@ def probs(self):
     def param_shape(self):
         return self._param.size()
 
+    def _log1pmprobs(self):
```

Comment:
Since it is a function for internal use, I think this can be moved to the top, like in MVN. Something like:

```python
def _log1pmtensor(tensor):
    # Do the same thing
```

Uses of the function in `kl.py` can be done via importing this function along with `Binomial`.